mcs/class/corlib/Test/System.Runtime.CompilerServices/TypeForwardedFromAttributeTest.cs

```diff
@@ -43,7 +43,7 @@ public void CtorTest ()
 			var a = new TypeForwardedFromAttribute ("System.Web, Version=2.0.0.0, Culture=Neutral, PublicKeyToken=b03f5f7f11d50a3a");
 			Assert.AreEqual ("System.Web, Version=2.0.0.0, Culture=Neutral, PublicKeyToken=b03f5f7f11d50a3a", a.AssemblyFullName);
 
-			Assert.Throws<ArgumentNullException> ( () => new TypeForwardedFromAttribute (null) );
+			var nullArgumentDoesntThrowException = new TypeForwardedFromAttribute (null);
```

Comment:
Hmm, we should followup with corefx why they changed this (it's still documented as throwing for null in https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.typeforwardedfromattribute.-ctor?view=netframework-4.7.1#Definition )

Reply:
@akoeplinger filed a bug to corefx 🙂 https://github.com/dotnet/corefx/issues/27554